src/languages/julia.js

```diff
@@ -84,7 +84,7 @@ function(hljs) {
 
   // placeholder for recursive self-reference
   var DEFAULT = {
-    aliases: ['jldoctest'],
+    aliases: ['julia.console'],
```

Comment:
> but looking around in highlight.js it seems that `.` is generally used for "scoping" related syntaxes together.

Huh? I wonder where did you find anything looking remotely like that :-) I'm certainly not aware of such a feature!